index.Rmd

```diff
@@ -196,21 +197,22 @@ par(col.axis="white",col.lab="white",tck=0)
 plotRGB(image2, r = 5, g = 4, b = 3, axes = TRUE, stretch = "lin", main = "False Color Composite")
 box(col="white")
 ```
+
```

Comment:
I think adding an extra line here will make it so  ###Indices of WATER doesn't show up next to the image in the html view.